frontend/integration-tests/tests/kubevirt/clone.vm.scenario.ts

```diff
@@ -0,0 +1,230 @@
+    }, VM_BOOTUP_TIMEOUT);
+
+    it('Cloned VM has cleared MAC addresses.', async() => {
+      await vm.addNic(networkInterface.name, networkInterface.mac, networkInterface.networkDefinition);
+      await vm.action('Clone');
+      await wizard.next();
+
+      const clonedVm = new VirtualMachine(`${vm.name}-clone`, vm.namespace);
+      leakedResources.add(JSON.stringify({name: clonedVm.name, namespace: clonedVm.namespace, kind: 'vm'}));
+      await clonedVm.navigateToTab(nicsTab);
+
+      await browser.wait(until.and(waitForCount($$('.co-resource-list__item'), 2)), PAGE_LOAD_TIMEOUT);
+      // TODO: Add classes/ids to collumn attributes so that divs can be easily selected
+      const mac = await resourceRows.first().$('div:nth-child(4)').getText();
+      expect(mac === '---').toBe(true, 'MAC address should be cleared');
+
+      await clonedVm.action('Delete');
+      leakedResources.delete(JSON.stringify({name: clonedVm.name, namespace: clonedVm.namespace, kind: 'vm'}));
+      await vm.removeNic(networkInterface.name);
+    }, VM_BOOTUP_TIMEOUT);
+
+    it('Cloned VM has vm.cnv.io/name label.', async() => {
+      await vm.action('Clone');
+      await wizard.next();
+
+      const clonedVm = new VirtualMachine(`${vm.name}-clone`, vm.namespace);
+      leakedResources.add(JSON.stringify({name: clonedVm.name, namespace: clonedVm.namespace, kind: 'vm'}));
+      expect(searchYAML(`vm.cnv.io/name: ${vm.name}`, clonedVm.name, clonedVm.namespace, 'vm')).toBe(true, 'Cloned VM should have vm.cnv.io/name label.');
+
+      await clonedVm.action('Delete');
+      leakedResources.delete(JSON.stringify({name: clonedVm.name, namespace: clonedVm.namespace, kind: 'vm'}));
+    }, VM_BOOTUP_TIMEOUT);
+
+    it('Clone VM with Container source.', async() => {
+      await vm.action('Clone');
+      await wizard.next();
+
+      const clonedVm = new VirtualMachine(`${vm.name}-clone`, vm.namespace);
+      leakedResources.add(JSON.stringify({name: clonedVm.name, namespace: clonedVm.namespace, kind: 'vm'}));
+      expect(searchYAML('kubevirt/cirros-registry-disk-demo', clonedVm.name, clonedVm.namespace, 'vm')).toBe(true, 'Cloned VM should have container image.');
+
+      await clonedVm.action('Start');
+      await clonedVm.action('Delete');
+      leakedResources.delete(JSON.stringify({name: clonedVm.name, namespace: clonedVm.namespace, kind: 'vm'}));
+    },VM_BOOTUP_TIMEOUT);
+
+    it('Clone VM with URL source.', async() => {
+      await urlvm.action('Start');
+      await urlvm.action('Clone');
+      await wizard.next();
+
+      const clonedVm = new VirtualMachine(`${urlvm.name}-clone`, urlvm.namespace);
+      leakedResources.add(JSON.stringify({name: clonedVm.name, namespace: clonedVm.namespace, kind: 'vm'}));
+      await clonedVm.action('Start');
+
+      // Verify cloned PVC exists
+      await browser.get(`${appHost}/k8s/ns/${testName}/persistentvolumeclaims`);
```

Comment:
you should also check that VM yaml has dataVolumeTemplate with `${clonedVm.name}-${urlvm.name}-rootdisk-clone` and none other.

Reply:
Done.